update/Pos/Update/Poll/Pure.hs

```diff
@@ -119,6 +119,7 @@ instance Bi UpdateProposal => MonadPoll PurePoll where
           appName = svAppName . upSoftwareVersion $ uProp
     deactivateProposal ui =
         PurePoll $ (Poll.psActiveProposals . at ui .= Nothing) >>
-                   (Poll.psActivePropsIdx . mapped . at ui .= Nothing)
+                   (Poll.psActivePropsIdx . mapped . at ui .= Nothing) >>
+                   (Poll.psActivePropsIdx %= HM.filter (not . null))
```

Comment:
Perhaps it would be more reasonable to use `do`-notation here instead of two sequenced `>>`s.